api/team_token_test.go

```diff
@@ -238,3 +238,26 @@ func (s *S) TestTeamTokenUpdate(c *check.C) {
 		},
 	}, eventtest.HasEvent)
 }
+
+func (s *S) TestTeamTokenInfo(c *check.C) {
+	newToken, err := servicemanager.TeamToken.Create(authTypes.TeamTokenCreateArgs{
+		Team:        s.team.Name,
+		Description: "desc",
+		TokenID:     "id1",
+	}, s.token)
+	c.Assert(err, check.IsNil)
+
+	request, err := http.NewRequest("GET", "/1.6/tokens/id1", nil)
+	c.Assert(err, check.IsNil)
+	request.Header.Set("Authorization", "bearer "+s.token.GetValue())
+	recorder := httptest.NewRecorder()
+	s.testServer.ServeHTTP(recorder, request)
+	c.Assert(recorder.Code, check.Equals, http.StatusOK)
+
+	token, err := servicemanager.TeamToken.FindByTokenID("id1")
```

Comment:
You don't need to check the token here. You could use `newToken` returned by `servicemanager.TeamToken.Create`